test/multiverse/lib/multiverse/suite.rb

```diff
@@ -266,6 +266,7 @@ def generate_gemfile(gemfile_text, env_index, local = true)
         f.puts "gem 'rake'" unless gemfile_text =~ /^\s*gem .rake[^_]./ || suite == 'rake'
 
         f.puts "gem 'mocha', '~> 1.9.0', :require => false"
+        f.puts "gem 'minitest-stub-const', '~> 0.6', require: false"
```

Comment:
This line is already included in the grpc branch. It was needed to have tests pass in this PR, so I've included it in this commit.